src/models/transformer_encoder/create_dataset_for_training_tokenizer.py

```diff
@@ -0,0 +1,39 @@
+from argparse import ArgumentParser
```

Comment:
Why is this file here and not in the `scripts` folder? There is no logic specific for model, just some data preparation